plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -600,14 +600,14 @@ function OPDSBrowser:showDownloads(item)
                 {
                     text = _("Page stream") .. "\u{2B0C}", -- append LEFT RIGHT BLACK ARROW
                     callback = function()
-                        self:streamPages(item, acquisition.href, acquisition.count)
+                        self:streamPages(acquisition.href, acquisition.count, false)
                         UIManager:close(self.download_dialog)
                     end,
                 },
                 {
                     text = _("Stream from page") .. "\u{2B0C}", -- append LEFT RIGHT BLACK ARROW
                     callback = function()
-                        self:streamPages(item, acquisition.href, acquisition.count, true)
+                        self:streamPages(acquisition.href, acquisition.count, true)
```

Comment:
Thank you, I've missed these calls and had no way to test the streaming.

Reply:
No Sweat, I figured it was a mistake.